src/lib.rs

```diff
@@ -3334,6 +3777,18 @@ static INTERFACE_SET: Lazy<Vec<InterfaceData>> = Lazy::new(|| {
     v
 });
 
+/// Provides access to the list of interfaces defined by this implementation
+///
+/// Starting with PKCS#11 version 3.0 tokens provide a list of interfaces
+/// that can be obtained from the token. Each interface provides a name, and
```

Comment:
I would say here `module` instead of token as the token is a term for the thing inside of the slot in pkcs11 terminology.
```suggestion
/// Starting with PKCS#11 version 3.0 modules provide a list of interfaces
/// that can be obtained from the module. Each interface provides a name, and
```